chia/consensus/blockchain.py

```diff
@@ -632,9 +632,9 @@ async def run_generator(self, unfinished_block: bytes, generator: BlockGenerator
         npc_result_bytes = await task
         if npc_result_bytes is None:
             raise ConsensusError(Err.UNKNOWN)
-        ret = NPCResult.from_bytes(npc_result_bytes)
+        ret: NPCResult = NPCResult.from_bytes(npc_result_bytes)
         if ret.error is not None:
-            raise ConsensusError(ret.error)
+            raise ConsensusError(Err(ret.error))
```

Comment:
Not necessarily a whole, just don't want to add extra unnecessary complexity in Streamable

Reply:
By making the class not describe what the non-serialized form is it means we 1) have to remember to convert it on every use and assignment and 2) have no parseable (code/hints) indication of what it is supposed to be.

Reply:
It's more of a security thing. Streamable was designed for converting a very small subset of objects into bytes so we can hash them into the blockchain. We have used streamable a little too much now to do all sorts of things, but I still think it's better to keep it simple and small in scope.

Reply:
Then we need a layer around it i guess if it can't even handle the error code type.  Maybe the enumeration could inherit from `enum.Enum` and also `uint16` to end up working.